src/encoded/static/components/home.js

```diff
@@ -641,16 +641,7 @@ export default class Home extends React.Component {
                 <div className="row">
                     <div className="col-xs-12">
                         <Panel>
-                            <AssayClicking assayCategory={this.state.assayCategory} handleAssayCategoryClick={this.handleAssayCategoryClick} />
-                            <HomeBanner adminUser={adminUser} />
-                            <div className="organism-tabs">
-                                <TabClicking organisms={this.state.organisms} handleTabClick={this.handleTabClick} />
-                            </div>
-                            <div className="graphs">
-                                <div className="row">
-                                    <HomepageChartLoader organisms={this.state.organisms} assayCategory={this.state.assayCategory} query={currentQuery} />
-                                </div>
-                            </div>
+                        <img src="/static/img/banner-bg.jpg  " alt="UT Southwestern" id="su-logo" height="390"/>
```

Comment:
banner-bg.jpg is not there. Or forget_ to copy in?

Reply:
Home page pic works now. But not full screen..
![image](https://user-images.githubusercontent.com/38848214/80785266-4ed7ef00-8b45-11ea-956e-e1d498c5fa84.png)